src/main/scala/com.snowplowanalytics.snowplow.badrows/Payload.scala

```diff
@@ -260,9 +260,38 @@ object Payload {
   }
 }
 
-/** Helper used to define a [[Payload.CollectorPayload]]. Isomoprhic to `org.apache.http.NaveValuePair` */
+/** Helper used to define a `CollectorPayload` and a `RawEvent`. Isomoprhic to `org.apache.http.NameValuePair` */
```

Comment:
I'd leave `Payload.CollectorPayload` and `RawEvent` in square brackets. Scalac uses them to then generate a link in ScalaDoc, but things like `org.apache.http.NameValuePair` cannot be linked because there're in 3rd-party jar.